public/js/actions/pause.js

```diff
@@ -126,7 +129,46 @@ function loadObjectProperties(grip) {
   };
 }
 
+/**
+ * Add expression for debugger to watch
+ * @param expression
+ */
+function addExpression(expression) {
+  return ({ dispatch, getState }) => {
+    dispatch({
+      type: constants.ADD_EXPRESSION,
+      id: expression.id || `${getExpressions(getState()).toSeq().size++}`,
+      input: expression.input
+    });
+  };
+}
+
+function updateExpression(expression) {
+  return ({ dispatch }) => {
+    dispatch({
+      type: constants.UPDATE_EXPRESSION,
+      id: expression.id,
+      input: expression.input
+    });
+  };
+}
+
+function evaluateExpressions() {
+  return ({ dispatch, getState, client }) => {
+    for (let expression of getExpressions(getState())) {
```

Comment:
It would be nice to have an `evaluateExpression` action, which can be called when an expression is updated.